rules_python/piptool.py

```diff
@@ -29,6 +29,9 @@
 # order, since these modules modify the import path and machinery.
 import pkg_resources
 
+WHEEL_CACHE_URL = "cached-python-packages.s3-website.us-east-1.amazonaws.com"
+PYPI_URL = "pypi.org/simple"
```

Comment:
This seems fine for testing, but IMO we should make this configurable and pass it in from `asana2`.  Look for how the command line args are passed in below.